tests/dummy/app/adapters/car.js

```diff
@@ -0,0 +1,23 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+import { AdapterMixin } from 'ember-custom-actions';
+
+export default JSONAPIAdapter.extend(AdapterMixin, {
+  urlForCustomAction(modelName, id, snapshot, requestType) {
+    let { adapterOptions: { suffix } } = snapshot;
+    suffix = suffix || '';
+
+    if (requestType === 'clean') {
+      let baseUrl = this._buildURL(modelName, id);
```

Comment:
the `_buildURL` is a private one and should be never considered to use in tests or public interfaces.